### PR TITLE
Update settings for header section

### PR DESCRIPTION
**Pull request by t-kelly**
### What are you trying to accomplish with this PR?
Fixes https://github.com/Shopify/slate/issues/172

For maintainers:
- [x] I have :tophat:'d these changes.
- [ ] I have bumped the `package.json` version in a separate PR, if applicable.

### CODE REVIEW

**Comment by chrisberthe** [comment]
Looks right - do you have a demo shop we could test?

File: src/sections/header.liquid [hunk 1]
```diff
@@ -121,10 +115,14 @@
...
       },
       {
-        "type": "text",
+        "type": "range",
         "id": "logo_max_width",
+        "min": 50,
+        "max": 450,
+        "step": 8,
+        "unit": "px",
         "label": "Custom logo width (in pixels)",
```
Comment (chrisberthe) [inline src/sections/header.liquid]: Units are there, so no need for `(in pixels)`.

File: src/sections/header.liquid [hunk 2]
```diff
@@ -121,10 +115,14 @@
         "label": "Logo image"
       },
       {
-        "type": "text",
+        "type": "range",
         "id": "logo_max_width",
+        "min": 50,
+        "max": 450,
+        "step": 8,
```
Comment (chrisberthe) [inline src/sections/header.liquid]: Is 8 a generic value you chose or one of our standards now?

Comment (chrisberthe) [inline src/sections/header.liquid]: Is 8 a generic value you chose or one of our standards now?

Comment (t-kelly) [inline src/sections/header.liquid]: Is there another theme that is using the range selector for the logo yet?

Comment (chrisberthe) [inline src/sections/header.liquid]: No, it seems like there aren't any yet for headers. I understood how you got 8 but was just wondering if we had established a pattern for this. Bump it to 10? 😛

**Comment by t-kelly** [comment]
> You'll need to add width: 100% on the image for it to scale properly.

Do we want to stretch the logo? Isn't that what would happen if we added `width: 100%;`?

File: src/sections/header.liquid (hunk 2 above)
Comment (t-kelly) [inline src/sections/header.liquid]: Sure 👍

**Comment by chrisberthe** [comment]
Not sure, realistically we wouldn't want to stretch the image. I just found it weird that with a smaller image it caps out at its actual width even if I'm selecting "400px". I'd be curious what others think.

**Comment by t-kelly** [comment]
> I'd be curious what others think.
@NathanPJF @MaxMusing @huguestennier ^^^

**Comment by huguestennier** [comment]
I feel like if we cap it then there's no point since it could just go smaller.

Hail mary, but isn't that against performance to load a big image for logo and make it smaller with the range slider? Shouldn't we encourage merchants to upload to the right size?

**Comment by huguestennier** [comment]
I feel like we should let it resize bigger even if it gets blurry because I wouldn't want the range slider to not work at some value since the image as a max-width.

The client can always upload a better image.

**Pull request merged**